core/kazoo_services/src/kz_services_quantities.erl

```diff
@@ -543,7 +544,8 @@ calculate_qubicle_recipient_updates(JObj, Updates) ->
     of
         'false' -> Updates;
         'true' ->
-            Key = [<<"qubicle">>, <<"recipients">>],
+            Offering = kz_json:get_ne_value([<<"qubicle">>, <<"recipient">>, <<"offering">>], JObj, <<"basic">>),
+            Key = <<Offering/binary, "_recipient">>,
```

Comment:
@mark2600 the key should be:

Key = [<<"qubicle">>, <<Offering/binary, "_recipient">>],